src/reducers/ComparisonSlice.ts

```diff
@@ -0,0 +1,28 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+interface IinitialState {
+  activeComparison: string;
+}
+
+const initialState: IinitialState = {
+  activeComparison: '',
```

Comment:
```suggestion
  activeComparison: 'All revisions',
```
This is the default I believe.